Assets/UI/Scripts/NewGameUI.gd

```diff
@@ -0,0 +1,44 @@
+extends Control
+
+var parent = null
+
+# Deactivate everything irrelevant for the time being
+func _ready() -> void:
+	var nodes = {
+		"Scenario": null,
+		"RandomMap": null,
+		"ResourceDensitySlider": null,
+		"Traders": null,
+		"Disasters": null
+	}
+
+	nodes = find_nodes(get_tree().get_root(), nodes)
+	#prints("nodes", nodes)
+	nodes["Scenario"].checkbox.disabled = true
+	nodes["RandomMap"].checkbox.disabled = true
+	nodes["ResourceDensitySlider"].editable = false
+	nodes["Traders"].disabled = true
+	nodes["Disasters"].disabled = true
+
+func find_nodes(root_node: Node, nodes_to_be_found: Dictionary) -> Dictionary:
+	for n in root_node.get_children():
+		if n.get_child_count() > 0:
+			find_nodes(n, nodes_to_be_found)
```

Comment:
Is this intentional? You're both changing the value in the argument and also returning it. It should probably only be one or the other, to avoid having two ways of writing the same code...

Reply:
I've removed the return part. Wasn't aware it passes the dictionary by reference.